src/toast-error/index.js

```diff
@@ -1,5 +1,6 @@
 import angular from 'angular';
 import translate from 'angular-translate';
+import translateAsyncLoader from '@ovh-ux/translate-async-loader';
```

Comment:
```suggestion
import '@ovh-ux/translate-async-loader';
```

In that way you can inject your AngularJS module by using `'translate-async-loader'`.